packages/website/pages/terms.js

```diff
@@ -23,75 +23,158 @@ export default function TermsOfService() {
+          expiration. In late 2022, we plan to build tooling to enable the 
+          non-custodial, perpetual renewal of storage contracts independent of 
+          {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} to secure the perpetual persistence of all NFTs without 
+          dependence on any single entity.
+        </p>
+        <p className="lh-copy">
+          In addition to storing data in Filecoin deals, {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} currently 
+          stores data on a self-managed IPFS Cluster with 6 nodes, with any given 
+          data replicated on 3 of them. These nodes are used to (1) store data 
+          while data is being aggregated into Filecoin deals, and (2) provide 
+          data to the public IPFS network for retrieval. This IPFS Cluster 
+          infrastructure may undergo changes over time, such as migrations or 
+          upgrades. Additional content-addressed copies of the data are also 
+          kept outside this IPFS Cluster.
+        </p>
+        <p className="lh-copy">
+          Data stored via {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} is guaranteed to be publicly available via 
+          IPFS, and users are responsible and liable for the data they upload. 
+          All data uploaded is available to anyone who requests it 
+          using the correct CID. Users should not store any private or sensitive 
+          information in an unencrypted form using {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '}. Further, deleting 
+          files from {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} will remove them from the file listing for a user's 
+          account, but nodes on the IPFS network may retain copies of the data indefinitely. 
+          Users should not use {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} for data that may need to be permanently 
+          deleted in the future.
+        </p>
+        <p className="lh-copy">
+          In a worst-case scenario where {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} needs to be terminated, the service 
+          will provide at least 180 days notice to users. Per the Filecoin protocol, all 
+          existing Filecoin deals will continue, independent of {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '}. {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} 
+          will do the following to make the transition for its users as seamless as possible: 
```

Comment:
What "transition" is being referred to here? If the service needs to be terminated?

Reply:
yeah. added a clause to clarify that this is referring to the hypothetical scenario in which the service had to be terminated